python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -267,6 +267,39 @@ def test_actor_method_bind_same_constant(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_actor_method_bind_same_arg(ray_start_regular):
```

Comment:
QQ: this is something intra process channel was already supposed to handle?

Reply:
That's right. I will clean it up.

Reply:
Done.